_test/test_sqw_class/test_sqw_signal.m

```diff
@@ -20,44 +20,60 @@
             obj.sqw_2d = read_sqw(test_sqw_2d_fullpath);
             obj.save();
         end
+
+        function test_signal_fb(obj)
+            clObj = set_temporary_config_options(hor_config, 'mem_chunk_size', 1000);
+
+            w1modE = coordinates_calc(obj.sqw_2d, 'E');
+
+            fb_sqw = obj.sqw_2d.copy();
+            fb_sqw.pix = PixelDataFileBacked(fb_sqw.pix);
+
+            assertTrue(fb_sqw.pix.is_filebacked);
+
+            w1modE_fb = coordinates_calc(obj.sqw_2d, 'E');
```

Comment:
I don't think this is filebacked, it is the same as L.27. Do you need fb_sqw as the first arg?

Reply:
You are correct, fixed.